pkg/controller/projectreference/projectreference_adapter.go

```diff
@@ -560,6 +570,25 @@ func (r *ReferenceAdapter) configureAPIS() error {
 	return nil
 }
 
+func (r *ReferenceAdapter) ConfigureProjectLabel() (gcputil.OperationResult, error) {
```

Comment:
we should not do this for CCS projects. I think when creating projects we don't set it, but this part may need a ccs check.

Reply:
Added isCCS check and fixed tests.